presentation/presentation.go

```diff
@@ -26,6 +28,15 @@ type uiContext struct {
 	prevReport   analysis.Report
 	cumulative   bool
 	paused       bool
+	useGui       bool
+	topX         uint16
+	maxVal       uint64
```

Comment:
This name is pretty ambiguous. I know it's "maximum number of values to report," but it could easily be interpreted as "maximum value seen." Please rename.

Reply:
It is the minimum threshold that sum(size) must be in order to be included in the report. Will change to minimumThreshold or minimumSizeThreshold.